interface/wx/intl.h

```diff
@@ -33,9 +33,25 @@ struct wxLanguageInfo
     /// language info structure.
     int Language;
 
+    /**
+        Tag for locale in BCP 47-like notation.
+
+        @since 3.1.6
+     */
+    wxString LocaleTag;
+
     /// Canonical name of the language, e.g. @c fr_FR.
     wxString CanonicalName;
 
+    /**
+        Canonical reference including region, if the name specifies
```

Comment:
Really a nitpick, but the brief description ought to end with a period for Doxygen, i.e.
```suggestion
        Canonical reference including region.
        
        If the name specifies
```

Reply:
Thanks. I'll try to keep this in mind for the next modifications.